src/app/shared/forms/utils/form-utils.ts

```diff
@@ -38,15 +47,52 @@ export function determineSalutations(countryCode: string): string[] {
       break;
     }
     case 'GB': {
-      salutationLabels = [
-        'account.salutation.ms.text',
-        'account.salutation.miss.text',
-        'account.salutation.mrs.text',
-        'account.salutation.mr.text',
-        'account.salutation.dr.text',
-      ];
+      salutationLabels = ['account.salutation.ms.text', 'account.salutation.mr.text', 'account.salutation.dr.text'];
       break;
     }
   }
   return salutationLabels;
 }
+
+/**
+ * Gets all possible salutation options for a certain country.
+ * @param translate instance of a translation service
+ * @param countryCode country code of the country for which the salutations should be determined.
+ * @returns salutation select options
+ */
+export function getSalutationOptionsForCountryCode(translate: TranslateService, countryCode: string): SelectOption[] {
```

Comment:
It seems bad practice to have a function you have to pass services to. Wouldn't it be nicer to create a form-utils service or similar that you can inject everywhere you use it? Because every piece of code that uses this function has to inject the translateService only to pass it along. It would be nicer to inject the right thing from the start.